sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/ScanProgress.kt

```diff
@@ -0,0 +1,28 @@
+package cash.z.ecc.android.sdk.internal.model
+
+internal data class ScanProgress(
+    private val numerator: Long,
+    private val denominator: Long
+) {
+    override fun toString() = "ScanProgress($numerator/$denominator) -> ${getSafeRation()}"
+
+    /**
+     * Returns progress ratio in [0, 1] range. Any out-of-range value is treated as 0.
+     */
+    fun getSafeRation() = numerator.toFloat().div(denominator).let { ration ->
```

Comment:
Should this be `Ratio` rather than `Ration`?

Reply:
Correct, good catch. Thanks!